packages/sit-onyx/src/styles/mixins/normalize.scss

```diff
@@ -0,0 +1,221 @@
+@mixin apply {
+  html:focus-within & {
+    scroll-behavior: smooth;
+  }
+
+  &,
+  * {
+    &,
+    &::before,
+    &::after {
+      margin: 0;
+
+      box-sizing: border-box;
```

Comment:
So we can remove this from all our existing components, right?

Reply:
correct!